PayzaarTest/ProductStorage.cs

```diff
@@ -0,0 +1,112 @@
+﻿namespace PayzaarTest
```

Comment:
ProductUtils was renamed to ProductStorage. The reason for this is because the class went from static to public, it seemed to me that a class has the functionality to show available products according to time and that has a list of available products acted like a Storage, hance the name, this behavior is best used when you are able to create instances. You can notice that I added the AddProduct and a RemoveProduct methods to the class in order for it acter better like a Storage.